Homework_Conditions/index.js

```diff
@@ -0,0 +1,130 @@
+        price: "37€",
+        rating: 4,
+        shippingCost: {
+            cost: freeShipping
+        }
+    },
+    {
+        imageUrl: "https://images-na.ssl-images-amazon.com/images/I/81Yq1G6O-CL._AC_UY445_.jpg",
+        name: "Women's Maxi Floral dress",
+        description: " Multicolored casual Floral dress with line",
+        price: "25€",
+        rating: 3,
+        shippingCost: {
+            cost: freeShipping
+        }
+    },
+    {
+        imageUrl: "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRF1avcP5Nli-Mu5mYvk1HvIhqu8QpFFumFonCHNNBwmuP9_1f4Qox_dJLTiM9ep58ZFTXEcHE&usqp=CAc",
+        name: "Women's Summer dress",
+        description: " Long Casual Elegant Dresses Party Dress ",
+        price: "35€",
+        rating: 5,
+        shippingCost: {
+            cost: freeShipping
+        }
+    },
+    {
+        imageUrl: "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRMancKgpx0vgr9eGIYERMONz6vWT-F5XuMgWmBnrjbOD04KCmQ0KYi-lEVsTrPe6hb264ZLrhU&usqp=CAc",
+        name: "Women's Nightie",
+        description: " Short,pleasantly loose cut ,100% cotton",
+        price: "15€",
+        rating: 4,
+        shippingCost: {
+            cost: freeShipping
+        }
+    }
+
+];
+let amountArray = [0, 1, 2, 3];
+
+let showCart = [];
+
+function dressListInput() {
+    for (let i = 0; i < dressList.length; i++) {
+        const dressElementDescr = document.createElement("div")
+        dressElementDescr.innerHTML = `<li> <h5>${dressList[i].name}</h5>
+    <img  class ="dressImage" src="${dressList[i].imageUrl}" />
+    <p>${dressList[i].description}</p>
+    <p>Price:${dressList[i].price}</p>
+    <span>${dressList[i].rating}</span>
+    <p>Delivery Cost :${dressList[i].shippingCost.cost.deliveryCharge}</p>
+    </li>`;
+        dressElementDescr.classList.add("dressDescription");
+        const myAmountList = document.getElementById("myAmount");
+        const newAmount = document.createElement("input");
+        newAmount.type = "number";
+        newAmount.classList.add("inputEle")
+        newAmount.placeholder = "Enter the required Amount";
+        myAmountList.appendChild(newAmount);
+        console.log(newAmount.value);
+        newAmount.addEventListener("change", function newValueSel() {
+            if (newAmount.value > amountArray[3]) {
+                document.getElementById("selectDress").innerHTML = "In Stock";
+            } else if (newAmount.value == amountArray[3] && newAmount.value > 1) {
+                document.getElementById("selectDress").innerHTML = "Almost Out of Stock";
+            } else if (newAmount.value == amountArray[1]) {
+                document.getElementById("selectDress").innerHTML = "Last Chance";
+            } else if (newAmount.value == amountArray[0]) {
+                document.getElementById("selectDress").innerHTML = "Out of Stock";
+            }
+        });
+        /* How can i add stars here to the property rating */
```

Comment:
Alright to add stars lets complete this `createStars` function. 

The function should return a string of html based on the number of stars argument.

example, assuming 5 is max rating, for the number 3 it should return something like 

```html
<span class="fa fa-star checked"></span>
<span class="fa fa-star checked"></span>
<span class="fa fa-star checked"></span>
<span class="fa fa-star unchecked"></span> 
<span class="fa fa-star unchecked"></span> 
```

```js
function createStars(starCount) {
  const maxHearts = 5; // the max rating possible
  const unFilledStars = maxHearts - starCount; //. the number of unchecked stars
  let html = ''; // using let because we are going to add to this variable 
  // loop through star count a create the html for check stars
  for (let i = 0; i < starCount; i++) {
    html += '<span class="fa fa-star checked"></span>';
  }

  for (let i = 0; i < unFilledStars; i++) {
   // TODO: add to the html unfilled / unchecked stars similar to how we added checked stars
  }
  return html;
}
```

then to use this function where you build out your product
```
<li>
...
${createStars(dressList[i].rating)}
...
</li>
```